scripts/spectrum-tokens.js

```diff
@@ -64,6 +64,7 @@ const tokenPackages = [
     'slider',
     'popover',
     'thumbnail',
+    'dropzone',
```

Comment:
@mlogsdon18 wanted to point this one piece out before I merged... even though there are no tokens acquired via the addition of this pattern to the list, adding it here ensures if we did add any later, it would be ready.